### PR TITLE
Update Service docs to 0.6.3 - Closes #1484

### LINKED ISSUE #1484: Update Service docs to v0.6.3
### What should be the new content?
See Lisk Service release notes: https://github.com/LiskHQ/lisk-service/releases

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1484

### CODE REVIEW

File: modules/ROOT/pages/configuration/docker.adoc [hunk 1]
```diff
@@ -53,6 +53,18 @@ GEOIP_JSON=
 # Create search index (0 = full index)
 INDEX_N_BLOCKS=0
 
+# Mysql Snapshot settings
```
Comment (nagdahimanshu) [inline modules/ROOT/pages/configuration/docker.adoc]:
```suggestion
# MySQL Snapshot settings
```

File: modules/ROOT/pages/configuration/docker.adoc [hunk 2]
```diff
@@ -53,6 +53,18 @@ GEOIP_JSON=
...
+
+# Enable or disable apply snapshot feature
+ENABLE_APPLY_SNAPSHOT=true
+# Custom snapshot url (Expected to end with sql.gz)
+INDEX_SNAPSHOT_URL='https://snapshots.lisk.io/mainnet/service.sql.gz'
+# When the MySql is hosted as a docker-compose service, set the following environment variables:
```
Comment (nagdahimanshu) [inline modules/ROOT/pages/configuration/docker.adoc]:
```suggestion
# When the MySQL is hosted as a docker-compose service, set the following environment variables:
```

File: modules/ROOT/pages/configuration/docker.adoc [hunk 3]
```diff
@@ -53,6 +53,18 @@ GEOIP_JSON=
...
+# Set docker-compose file path (use absolute path)
+DOCKER_COMPOSE_FILEPATH='/Users/lisk/lisk-service/jenkins/mysql/docker-compose.yml'
+# Set mysql service name as defined in the above docker-compose file
```
Comment (nagdahimanshu) [inline modules/ROOT/pages/configuration/docker.adoc]:
```suggestion
# Set MySQL service name as defined in the above docker-compose file
```

File: modules/ROOT/pages/reference/configuration.adoc [hunk 4]
```diff
@@ -117,6 +117,26 @@ LISK_CORE_WS=ws://localhost:8080       # Lisk Core WebSocket RPC API
 LISK_CORE_CLIENT_TIMEOUT=30            # Lisk Core client timeout (in seconds)
 ----
 
+=== Mysql Snapshot settings
```
Comment (nagdahimanshu) [inline modules/ROOT/pages/reference/configuration.adoc]:
```suggestion
=== MySQL Snapshot settings
```

File: modules/ROOT/pages/reference/configuration.adoc [hunk 5]
```diff
@@ -117,6 +117,26 @@ LISK_CORE_WS=ws://localhost:8080       # Lisk Core WebSocket RPC API
...
+
+[source,bash]
+----
+# Enable or disable apply snapshot feature
+ENABLE_APPLY_SNAPSHOT=true
+
+# Custom snapshot url (Expected to end with sql.gz)
+INDEX_SNAPSHOT_URL='https://snapshots.lisk.io/mainnet/service.sql.gz'
+
+# When the MySql is hosted as a docker-compose service, set the following environment variables:
```
Comment (nagdahimanshu) [inline modules/ROOT/pages/reference/configuration.adoc]:
```suggestion
# When the MySQL is hosted as a docker-compose service, set the following environment variables:
```

File: modules/ROOT/pages/reference/configuration.adoc [hunk 6]
```diff
@@ -117,6 +117,26 @@ LISK_CORE_WS=ws://localhost:8080       # Lisk Core WebSocket RPC API
...
+# Set docker-compose file path (use absolute path)
+DOCKER_COMPOSE_FILEPATH='/Users/lisk/lisk-service/jenkins/mysql/docker-compose.yml'
+
+# Set mysql service name as defined in the above docker-compose file
```
Comment (nagdahimanshu) [inline modules/ROOT/pages/reference/configuration.adoc]:
```suggestion
# Set MySQL service name as defined in the above docker-compose file
```

**Review by nagdahimanshu — CHANGES_REQUESTED** [review]
Looks good overall apart from one minor change needed.

**Review by chris529 — APPROVED** [review]
LGTM 👍

**Review by nagdahimanshu — APPROVED** [review]
LGTM 👍

**Pull request merged**